src/components/ColumnMenuButton/ColumnMenuButton.module.css

```diff
@@ -0,0 +1,32 @@
+.columnMenuButton {
```

Comment:
@severo thoughts on semantic selectors here?

Reply:
first: it should go to HighTable/HighTable.module.css (we have only one global CSS module for the library, it's the simplest way we found to make it easy to override the styles if needed).

re semantic selectors: @rembrandtreyes already fixed it!